Source/SwiftLintFramework/Rules/Lint/PrivateUnitTestRule.swift

```diff
@@ -106,12 +106,48 @@ public struct PrivateUnitTestRule: SwiftSyntaxRule, ConfigurationProviderRule, C
                 private ↓func test4() {}
             }
             """)
+        ],
+        corrections: [
+            Example("""
+
+                private class Test: XCTestCase {}
```

Comment:
Are these empty newlines in these examples intentional?

Reply:
Yes. In the example
```swift
class Test {
    private fund test() {}
}
```
the `private` token has leading trivia that must be preserved when removing it by attaching the trivia to the next token. This is what the unpleasant `withoutPrivate(modifiers:declKeyword:)` method does. The empty lines in the tests make sure this works.

If there is a better way to deal with the whitespace, please let me know.